notebooks/community/explainable_ai/SDK_Custom_Container_XAI.ipynb

```diff
@@ -0,0 +1,1283 @@
+{
```

Comment:
<p>Each code cell should have text cell explaining what it does</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/564/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>